near-sdk-macros/src/core_impl/code_generator/attr_sig_info.rs

```diff
@@ -75,7 +75,7 @@ impl AttrSigInfo {
     /// and `SUBTYPE` is one of the following: `[T; n]`, path like
     /// `std::collections::HashMap<SUBTYPE, SUBTYPE>`, or tuple `(SUBTYPE0, SUBTYPE1, ...)`.
     /// # Example
-    /// ```ignore
+    /// ```rust
     /// struct Input {
```

Comment:
@akorchyn can you kill all the boxes with checkmarks? 
and this one https://github.com/near/near-sdk-rs/issues/1288#issuecomment-2564292206

![image](https://github.com/user-attachments/assets/5a0087ba-594e-438a-bbb3-d60cf1f895a1)

Reply:
Well, I think the proper moment to click a checkmark it's when it merged. But Yeah, I cover all of them.